app/views/subnets/index.html.erb

```diff
@@ -26,9 +26,11 @@
         <td><%= link_to_if_authorized(hosts_count[subnet], hash_for_hosts_path(:search => "subnet.name=\"#{subnet}\"")) %>
         <td class="col-md-1">
           <%= action_buttons(display_delete_if_authorized(
-            hash_for_subnet_path(:id => subnet).
-            merge(:auth_object => subnet, :authorizer => authorizer),
-            :data => { :confirm => _("Delete %s?") % subnet.name })) %>
+                               hash_for_subnet_path(:id => subnet).
+                               merge(:auth_object => subnet, :authorizer => authorizer),
+                               :data => { :confirm => _("Delete %s?") % subnet.name }),
+                             render_pagelets_for(:action_buttons, subject: subnet)
```

Comment:
I think `action_buttons` is not a good symbol to name a pagelet. There are many "action buttons" in the app, this should have been something like `subnet_index_action_buttons`. Would you mind opening another core PR (just do refs issue number) fixing this while we can?